packages/build/src/plugins_core/blobs_upload/index.ts

```diff
@@ -72,7 +72,7 @@ const coreStep: CoreStepFunction = async function ({
         systemLog(`Uploading blob ${key}`)
 
         const { data, metadata } = await getFileWithMetadata(key, contentPath, metadataPath)
-        await blobStore.set(key, data, { metadata })
+        await blobStore.set(key, data.buffer, { metadata })
```

Comment:
Looks like it fails on node 14 😞

Reply:
NodeJS 14 supports Buffer.prototype.buffer - not sure why it would fail, I see some NodeJS 14 CI jobs passing and some others failing

Reply:
We only run one set of tests on ubuntu-latest + node v14, so the other tests you see passing are just a different block of tests.

Reply:
Looking at the test output, it might just be a matter of updating the tests as well?
`t.is(blob1.data, 'some value') ->   t.is(blob1.data.buffer, 'some value')`

Reply:
@mrstork using NodeJS 14.17 instead of 14.16 works - do we need to support 14.16?

Reply:
The range selection is before my time, I know it's not trivial to reduce the range though